File: 2021/May/Day 14/height-checker.py
# Source :https://leetcode.com/problems/height-checker/submissions/
class Solution(object):
    def heightChecker(self, heights):
        """
        :type heights: List[int]
        :rtype: int
        """

        copy = sorted(heights)

        count =0;
        for i in range(len(heights)):
            if heights[i]!=copy[i]:
                count+=1
        return count

File: 2023/June/Day 5/heightChecker.cpp
// Source : https://leetcode.com/problems/height-checker/
class Solution {
public:
    int heightChecker(vector<int>& heights) {
        vector<int> sortedHeights = heights;
        sort(sortedHeights.begin(),sortedHeights.end());
        
        int count=0;
        for(int i=0;i<size(heights);i++)
        {
            if(heights[i]!=sortedHeights[i])
            {
                count++;
            }
        }
        return count;
    }
};